.codex/skills/sdd-init/SKILL.md

```
@@ -0,0 +1,116 @@
+---
+name: sdd-init
+description: sdd-slice-wishで決めた「推奨スライス」を入力として、仕様書 docs/specs/YYYYMMDD-{name}.md のドラフトを作成する。ここでは実装もテスト実装も行わない。テストケースの網羅（境界・異常・不変条件の列挙）は sdd-test-cases の責務。
+---
+
+# sdd-init（Specドラフト作成）
+
+## 目的
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)